packages/edit-site/src/components/list/added-by.js

```diff
@@ -0,0 +1,184 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalHStack as HStack,
+	Icon,
+	Tooltip,
+} from '@wordpress/components';
+import { useRefEffect } from '@wordpress/compose';
+import { store as coreStore } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+import { useState } from '@wordpress/element';
+import { layout as themeIcon, plugins as pluginIcon } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+const TEMPLATE_POST_TYPE_NAMES = [ 'wp_template', 'wp_template_part' ];
+
+function useOnImageLoads( onLoad ) {
+	return useRefEffect( ( node ) => {
+		node.addEventListener( 'load', onLoad );
+
+		return () => {
+			node.removeEventListener( 'load', onLoad );
+		};
+	}, [] );
```

Comment:
`onLoad` should be in the dependencies. I'm not sure why we need this? Couldn't we just use `onLoad` attribute on `<img>`?

Reply:
Good point! I'll update it.